src/test/scala/io/citrine/lolo/bags/BaggerTest.scala

```diff
@@ -160,6 +189,21 @@ object BaggerTest {
     */
   def main(argv: Array[String]): Unit = {
     new BaggerTest()
-      .testInterrupt()
+      .testUncertaintyCalibration()
+  }
+
+
+  def getStandardRMSE(testSet: Seq[(Vector[Any], Double)], model: BaggedModel): Double = {
+    val predictions = model.transform(testSet.map(_._1))
+    val pva = testSet.map(_._2).zip(
+      predictions.getExpected().asInstanceOf[Seq[Double]].zip(
+        predictions.getUncertainty().get.asInstanceOf[Seq[Double]]
+      )
+    )
+    val standardError = pva.map{ case (a: Double, (p: Double, u: Double)) =>
+      Math.abs(a - p) / u
```

Comment:
Is there a case where u could be equal to zero?  If so, we should set a minimum value for u here.

Reply:
If `u == 0`, its an error.